WireAPI/Tests/WireAPITests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -30,7 +30,10 @@ final class AuthenticationManagerTests: XCTestCase {
     override func setUpWithError() throws {
         cookieStorage = MockCookieStorageProtocol()
         backendURL = try XCTUnwrap(URL(string: "https://www.example.com"))
-        let networkService = NetworkService(baseURL: backendURL)
+        let networkService = NetworkService(
+            baseURL: backendURL,
+            serverTrustValidator: ServerTrustValidator(pinnedKeys: [])
```

Comment:
Is there a way to test the ServerTrustValidator as well ? I'm assuming no, just wondering

Reply:
Yes, I've just added a tests for that

Reply:
I think I will also test `BackendService`